src/components/FeaturedListing/Card/Card.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
+
+import colors from 'styles/colors';
+import { Tag } from './card.styles';
+
+describe('test styled components that have props', () => {
+  it('should apply default styles to Tag component', () => {
+    // eslint-disable-next-line react/jsx-filename-extension
+    const tree = renderer.create(<Tag />).toJSON();
```

Comment:
We only use `enzyme` wrapper native functions, and we don't test css properties. This could be valid only in extreme/rare cases and not on our day to day testing.

please only use `shallow`, `mount` or `render` ..etc and test on the returned wrapper/container.
```suggestion
    const wrapper = shallow(<Tag />);
    wrapper.find(...etc)
```